metadata_backend/server.py

```diff
@@ -92,6 +92,7 @@ async def init() -> web.Application:
         web.get("/users/{userId}", _user.get_user),
```

Comment:
just a wild idea, how about we add query string parameters to this endpoint so that we can do something like:
`GET /users/current?type=drafts` i put `type` for lack of inspiration but the idea is to list both `folders` and `drafts` even though for folders it will be just a list of strings

Reply:
I have named this parameter `items` now, so `/users/current?items=drafts` returns the paginated list of drafts from the user object and `/users/current?items=folders` returns the list of folder IDs from the user object.